invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordManagement.js

```diff
@@ -27,28 +26,29 @@ export const RecordManagement = (props) => {
   return (
     <Grid relaxed>
       <Grid.Column>
-        <Grid.Row>
-          <Icon name="cogs" />
-          <h2 id="record-management-header">{i18next.t('Manage')}</h2>
-        </Grid.Row>
         <Grid.Row className="record-management-row">
           <EditButton recid={recid} onError={handleError} />
+        </Grid.Row>
+        <Grid.Row className="record-management-row">
           <NewVersionButton
+            style={{ display: "block" }}
```

Comment:
I agree, I mainly used it because it's how the [`NewVersionButton`](https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/components/NewVersionButton.js#L41) was implemented.
To fix this we will need to refactor it, I will create an issue as this tasks scope already grew a lot.